src/components/ha-service-control.ts

```diff
@@ -341,10 +343,9 @@ export class HaServiceControl extends LitElement {
                     .selector=${dataField.selector}
                     .key=${dataField.key}
                     @value-changed=${this._serviceDataChanged}
-                    .value=${this._value?.data &&
-                    this._value.data[dataField.key] !== undefined
+                    .value=${this._value?.data
                       ? this._value.data[dataField.key]
-                      : dataField.default}
```

Comment:
Let's add `dataField.default` as a `placeholder`, so the user can see what the default is when he leaves it empty

Reply:
added